src/components/tx/TxRow.js

```diff
@@ -9,15 +9,60 @@ import React from 'react';
 import { useHistory } from 'react-router-dom';
 import hathorLib from '@hathor/wallet-lib';
 import dateFormatter from '../../utils/date';
+import { useIsMobile, useNewUiEnabled } from '../../hooks';
 
-function TxRow({ tx }) {
+const TxRow = ({ tx }) => {
+  const newUiEnabled = useNewUiEnabled();
+  const isMobile = useIsMobile();
   const history = useHistory();
 
   const handleClickTr = hash => {
     history.push(`/transaction/${hash}`);
   };
 
-  return (
+  const parseTimestamp = timestamp => {
+    const date = new Date(timestamp * 1000);
+
+    const userLocale = navigator.language || navigator.userLanguage || 'en-US';
+
+    return new Intl.DateTimeFormat(userLocale, {
+      year: 'numeric',
+      month: '2-digit',
+      day: '2-digit',
+      hour: '2-digit',
+      minute: '2-digit',
+      second: '2-digit',
+      hour12: false,
+    })
+      .format(date)
+      .replace(',', '');
+  };
+
+  const renderIdCell = id => {
+    const idStart = id.substring(0, isMobile ? 4 : 12);
+    const idEnd = id.substring(id.length - (isMobile ? 4 : 12), id.length);
+
+    return (
+      <div className="id-cell">
+        {idStart}
+        <div className="ellipsis">
+          <div className="ellipsi"></div>
+          <div className="ellipsi"></div>
+          <div className="ellipsi"></div>
+        </div>
+        {idEnd}
+      </div>
+    );
+  };
```

Comment:
suggestion: Maybe we should move this to the utils folder so we can use this timestamp parser in other places as well.

Reply:
We moved it, in the next PR it will be in the utils/date.js